### PR TITLE
:sparkles: expenses / submit image by paste (ctrl+v)

### PR DESCRIPTION
**Pull request by guillaumecrespel**
Paste and submit image on web application

### CODE REVIEW

File: ui/ui-expenses/src/screens/list/list.reactions.js [hunk 1]
```diff
@@ -1,9 +1,32 @@
+/* eslint no-plusplus: ["error", { "allowForLoopAfterthoughts": true }] */
 import pica from 'pica/dist/pica'
 
+export const addListener = (action, store, { window }) => {
```
Comment (guillaumecrespel) [inline ui/ui-expenses/src/screens/list/list.reactions.js]: map event listener with k-ramel screens events

File: ui/ui-expenses/src/screens/list/list.reactions.js [hunk 2]
```diff
@@ -1,9 +1,32 @@
...
+  window.addEventListener('paste', pasteEvent => store.dispatch({ type: '@@ui/ON_PASTE', payload: pasteEvent }))
+}
+
+export const removeListener = (action, store, { window }) => {
+  window.removeEventListener('paste')
```
Comment (fabienjuif) [inline ui/ui-expenses/src/screens/list/list.reactions.js]: You have to remove the function: https://developer.mozilla.org/fr/docs/Web/API/EventTarget/removeEventListener

File: ui/ui-expenses/src/screens/list/list.reactions.js [hunk 3]
```diff
@@ -1,9 +1,32 @@
...
+
 export const load = (action, store, { http }) => {
   http('EXPENSES').get('/api/expenses')
 }
 
+export const submitClipboardImage = async ({ payload }, store) => {
+  if (!payload || !payload.clipboardData || !payload.clipboardData.items) return
+
+  const { items } = payload.clipboardData
+  for (let i = 0; i < items.length; i++) { // items it's not an array
```
Comment (fabienjuif) [inline ui/ui-expenses/src/screens/list/list.reactions.js]: Ok, then what it is ?

File: ui/ui-expenses/src/screens/list/list.reactions.js [hunk 4]
```diff
@@ -1,3 +1,30 @@
+/* eslint no-plusplus: ["error", { "allowForLoopAfterthoughts": true }] */
```
Comment (fabienjuif) [inline ui/ui-expenses/src/screens/list/list.reactions.js]: can you remove this please.
you can write `i += 1`

**Pull request merged**